oldScripts/README.md

```diff
@@ -0,0 +1,27 @@
+# README
+
+### 1. Environment Setup
+- Store GradeScope credentials in environment variables `EMAIL` and `PASSWORD`. These can be defined inside a `.env` file.
+
+### 2. Google Authentication
+- Follow the steps at the following link to enable Google authentication: `"https://developers.google.com/sheets/api/quickstart/python#authorize_credentials_for_a_desktop_application"`
```

Comment:
Please edit this again later in the other PR 

https://github.com/AFA-Tooling/GradeSync/pull/13/files

because this Google Authentication API is deprecated and blocks automation for the gradescope to sheets API. 
`      creds = flow.run_local_server(port=0)`

Reply:
I will approve this for now, but you should change this README later once the Authentication method has been refactored.

Reply:
Will do!

Reply:
After the token.json file is created but the method the first time it is run, user interaction is no longer needed. However, you have a point in that we should consider approaching Google authentication differently.